internal/core/data/router_test.go

```diff
@@ -184,7 +182,10 @@ func TestRestValueDescriptorsUsageHandler(t *testing.T) {
 			dbClient = tt.dbMock
 			Configuration.Service = tt.config
 			rr := httptest.NewRecorder()
-			handler := http.HandlerFunc(restValueDescriptorsUsageHandler)
+			handler := http.HandlerFunc(func(writer http.ResponseWriter, request *http.Request) {
```

Comment:
I wonder if this series of tests shouldn't be modified to call restValueDescriptorsUsageHandler() directly.  Beyond goosing code coverage, I don't see any value going through the full HTTP request/response cycle.